ArmVirtPkg/ArmVirt.dsc.inc

```diff
@@ -150,11 +150,6 @@
   # CryptoPkg libraries needed by multiple firmware features
   #
   IntrinsicLib|CryptoPkg/Library/IntrinsicLib/IntrinsicLib.inf
-!if $(NETWORK_TLS_ENABLE) == TRUE

```

Comment:
NIT: I think this could be left as the default libraries to use for ARM. Only specifying the accel libraries for the AARCH64 build should work.